types/tgwf__co2/index.d.ts

```diff
@@ -0,0 +1,14 @@
+declare namespace co2js {
+    export { co2 };
+    export { hosting };
+    export { averageIntensity };
+    export { marginalIntensity };
+}
+
+import co2 from "./co2.d.ts";
+import averageIntensity from "./data/average-intensities.min.d.ts";
+import marginalIntensity from "./data/marginal-intensities-2021.min.d.ts";
+import hosting from "./hosting.d.ts";
+export { averageIntensity, co2, hosting, marginalIntensity };
+
+export default co2js;
```

Comment:
unfortunately, DT still technically only provides types for commonjs -- so you need to use `export = co2js` here, which is the Typescript syntax for commonjs exports. `export default co2js` is only allowed for commonjs when the commonjs version exports a default property like so: `require('@tgwf/co2').default`. But @tgwf/co2 doesn't do this.
```suggestion
export = co2js;
```

You can hack in types for ES exports, but it's still janky. See `types/three` for an example.